pkgs/top-level/all-packages.nix

```diff
@@ -32643,6 +32643,7 @@ with pkgs;
   inherit (callPackages ../development/libraries/wlroots {})
     wlroots_0_15
```

Comment:
Can we upgrade wlroots everywhere to drop wlroots_0_15 ?

Reply:
tracking issue: https://github.com/NixOS/nixpkgs/issues/203222

The last obstacle is that hikari still has not released a version adapted to wlroots 0.16